Source/OxyPlot/Series/BarSeries/ErrorBarSeries.cs

```diff
@@ -110,17 +110,8 @@ protected internal override void UpdateMaxMin()
                 }
             }
 
-            var valueAxis = this.GetValueAxis();
-            if (valueAxis.IsVertical())
```

Comment:
Was this a hold-over from when Column/Bar series were separate concerns?

Reply:
Yes, exactly